docs/getting-started/installation-and-upgrade/other-installation-methods/rancher-behind-an-http-proxy/install-kubernetes.md

```diff
@@ -15,6 +15,84 @@ export HTTPS_PROXY=http://${proxy_host}
 export NO_PROXY=127.0.0.0/8,10.0.0.0/8,cattle-system.svc,172.16.0.0/12,192.168.0.0/16
 ```
 
+<Tabs>
+<TabItem value="K3s">
+
+First configure the HTTP proxy settings on the K3s systemd service, so that K3s's Containerd can pull images through the proxy:
+
+```
+cat <<'EOF' | sudo tee /etc/default/k3s > /dev/null
+HTTP_PROXY=http://${proxy_host}
+HTTPS_PROXY=http://${proxy_host}"
+NO_PROXY=127.0.0.0/8,10.0.0.0/8,cattle-system.svc,172.16.0.0/12,192.168.0.0/16,.svc,.cluster.local
+EOF
+```
+
+Then run the K3s installation script to create a new K3s cluster. Ensure that the K3s version you are installing is [supported by Rancher](https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/).
+
+On the first node, create a new cluster:
+```
+curl -sfL https://get.k3s.io | INSTALL_K3S_CHANNEL=v1.23 K3S_TOKEN=your_secret sh -s - server --cluster-init
```

Comment:
How about `INSTALL_K3S_CHANNEL=v1.xx`? This will also fail, when you copy & paste it, and it makes it easier to see where you have to change something.

Reply:
Sure, let's go with that.

Reply:
I updated it.